app/src/main/java/com/kickstarter/ui/viewholders/ErroredBackingViewHolder.kt

```diff
@@ -0,0 +1,62 @@
+package com.kickstarter.ui.viewholders
+
+import android.view.View
+import com.jakewharton.rxbinding.view.RxView
+import com.kickstarter.libs.RelativeDateTimeOptions
+import com.kickstarter.libs.rx.transformers.Transformers.observeForUI
+import com.kickstarter.libs.utils.DateTimeUtils
+import com.kickstarter.libs.utils.ObjectUtils.requireNonNull
+import com.kickstarter.models.ErroredBacking
+import com.kickstarter.viewmodels.ErroredBackingViewHolderViewModel
+import kotlinx.android.synthetic.main.item_errored_backing.view.*
+import org.joda.time.DateTime
+
+class ErroredBackingViewHolder(private val view: View, val delegate: Delegate?) : KSViewHolder(view) {
+
+    interface Delegate {
+        fun managePledgeClicked(projectSlug: String)
+    }
+
+    private val ksString = environment().ksString()
+    private var viewModel = ErroredBackingViewHolderViewModel.ViewModel(environment())
+
+    init {
+
```

Comment:
Should we have an extra space here?

Reply:
I actually don't know lol. Not sure if we have a style guideline for spaces after opening brackets.

Reply:
Ok, I normally like to have a standard about open bracket spacing for consistency but not a blocker